frontend/packages/core/src/NPS/header.tsx

```diff
@@ -58,19 +58,24 @@ export const generateFeedbackTypes = (workflows: Workflow[]): SelectOption[] =>
   workflows.forEach(workflow => {
     const { group, path, routes, displayName } = workflow;
 
-    if (!typeMap[group]) {
-      typeMap[group] = [];
-    }
-
     routes.forEach(route => {
-      typeMap[group].push({
-        label: route.displayName || displayName,
-        value: `/${path}/${route.path}`.replace(/\/\/+/g, "/"),
-      });
-
       const additionalNPS = get(route, "componentProps.additionalNPS", []);
-      if (additionalNPS) {
-        typeMap[group].push(...additionalNPS);
+
+      if (route.hideNav === undefined || route.hideNav === false || additionalNPS.length > 0) {
+        if (!typeMap[group]) {
+          typeMap[group] = [];
+        }
+
+        if (route.hideNav === undefined || route.hideNav === false) {
```

Comment:
super nit
```suggestion
      const showRoute = route.hideNav === undefined || route.hideNav === false;
      if (showRoute || additionalNPS.length > 0) {
        if (!typeMap[group]) {
          typeMap[group] = [];
        }

        if (showRoute) {
```

Reply:
Haha that's so much nicer, I was going through if logic hell when I wrote it :)